docs/docs-content/automation/palette-cli/commands/vmo.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_label: "VMO"
+title: "VMO"
+description: "Reference resource for the vmo command."
+hide_table_of_contents: false
+sidebar_position: 60
+tags: ["palette-cli"]
+---
+
+Use the `vmo` command to migrate Virtual Machines (VMs) and import and deploy vSphere Open Virtual Appliances (OVAs).
+The VMs can then be used with the Virtual Machine Orchestrator (VMO).
+
+## Subcommands
+
+- [`deploy-ova`](#deploy-ova) - Deploy an imported vSphere OVA. This command requires you to have an OVA deployment
+  file. If you do not have one, you can generate an OVA with the `import-ova` subcommand.
+
+- [`import-ova`](#import-ova) - Import a vSphere OVA. This subcommand will generate an OVA deployment configuration
+  file. The configuration can then be deployed using the `deploy-ova` subcommand.
+
+- [`migrate-vm`](#migrate-vm) - Migrate one or more VMware vSphere VMs to Palette VMO.
+
+## Prerequisites
+
+- A Healthy VMO cluster. Refer to the [Create a VMO Profile](../../../vm-management/create-vmo-profile.md) for further
+  guidance.
+- One or more VMs hosted in VMware vSphere. The VMs should also operate one the
+  [`virt-v2v` supported guest systems](https://libguestfs.org/virt-v2v-support.1.html).
+
+## Limitations
+
+- You can only use the `vmo` subcommand with VMs hosted in VMware vSphere.
```

Comment:
Limitations per the CLI:

```
Prerequisites:
  - VMware credentials are required. Migration can be performed with vCenter credentials only, but it can be
    optionally accelerated by providing credentials for the ESXi host(s) where the VM(s) reside.
  - VMs must be powered off. For Windows VMs, shut down at the virtualized OS level - not from the vSphere console.
  - If migrating more than one VM in the same plan, all VMs must share the same network.
  - Not all guest OSs are supported. See https://github.com/libguestfs/virt-v2v/blob/v2.3.5/docs/virt-v2v-support.pod
    for a complete support matrix.
```

We should call out the note about powering off Windows VMs.

The fact that choosing ESXi mode accelerates the data transfer with VDDK is also potentially important.